packages/lib-classifier/src/store/SubjectViewerStore/SubjectViewerStore.js

```diff
@@ -67,20 +71,12 @@ const SubjectViewer = types
     }
 
     return {
-      afterRootInitialize () {
-        // afterRootInitialize() enables listening on store.workflowSteps because they are both now initialized
-        reaction(
-          () => getRoot(self)?.workflowSteps.hasAnnotateTask,
-          (val) => {
-            self.setAnnotateVisibility(val)
-          }
-        )
-        
-        // reactions handle any changes to state, not the initial state
-        self.setAnnotateVisibility(getRoot(self)?.workflowSteps.hasAnnotateTask)
-      },
-      
       afterAttach () {
+        function _syncAnnotateVisibility() {
+          console.log('syncing annotate visibility', self.hasAnnotateTask)
+          self.setAnnotateVisibility(self.hasAnnotateTask)
```

Comment:
`self.annotateVisibility` is kind of redundant here, as it mirrors the value of `self.hasAnnotateTask`.

Reply:
In fact, you could rename the `hasAnnotateTask` view to `annotateVisibility`, and remove this reaction altogether. BUt then, you wouldn't be able to set `subjectViewer.annotateVisibility` from outside the model, which might be important.